front/app/components/SignUpIn/SignUp/PasswordSignup.tsx

```diff
@@ -173,19 +175,26 @@ class PasswordSignup extends PureComponent<Props & InjectedIntlProps, State> {
 
     this.setState({ token: metaData?.token });
 
+    const { onError, intl } = this.props;
+
     if (metaData?.token) {
       request<IUser>(
         `${API_PATH}/users/by_invite/${metaData.token}`,
         null,
         { method: 'GET' },
         null
-      ).then((response) => {
-        this.setState({
-          firstName: response?.data?.attributes?.first_name || null,
-          lastName: response?.data?.attributes.last_name || null,
-          email: response?.data?.attributes?.email || null,
+      )
+        .then((response) => {
+          this.setState({
+            firstName: response?.data?.attributes?.first_name || null,
+            lastName: response?.data?.attributes.last_name || null,
+            email: response?.data?.attributes?.email || null,
+          });
+        })
+        .catch(() => {
```

Comment:
This error handling was first done in `SignUp/index.tsx` by making another request just to check if it would return an error, while this request would not be checked for errors. Now it is making only one request, and it checks the error of that. Makes a lot more sense IMO